CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ endif(IS_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/.git)
 find_package(Threads REQUIRED)
 set(THREAD_LIBRARIES ${CMAKE_THREAD_LIBS_INIT})
 
-find_package(Boost 1.39.0 REQUIRED COMPONENTS program_options filesystem system )
+find_package(Boost 1.39.0 REQUIRED COMPONENTS program_options unit_test_framework filesystem system )
```

Comment:
I think we should only look for `unit_test_framework` when testing is enabled.

Reply:
Good idea.

Reply:
There is one other problem. `libvotca_tools` should be linked against `libboost_unit_test_framework` as it doesn't use any symbol from that library! This is to prevent over-linking.

So we need to split `libboost_unit_test_framework` out of `Boost_LIBRARIES`, which `libvotca_tools` is linked against.

Reply:
Ok, see the commit is just pushed.

Reply:
Thanks for doing that I would have had a hard time figuring that out.